tests/antithesis/test-template/robustness/traffic/main.go

```diff
@@ -43,7 +43,8 @@ var (
 		MinimalQPS:                     100,
 		MaximalQPS:                     1000,
 		BurstableQPS:                   1000,
-		ClientCount:                    3,
+		MemberClientCount:              3,
```

Comment:
I don't recommend mixing refactors and feature changes. Can you send a separate PR?